liquidator/liquidator.js

```diff
@@ -118,70 +112,82 @@ class Liquidator {
+      Logger.debug({
         at: "liquidator",
-        message: "potential withdrawable liquidations detected!💰",
-        number: potentialWithdrawableLiquidations.length,
-        potentialWithdrawableLiquidations: potentialWithdrawableLiquidations
+        message: "attempting to withdraw rewards from liquidations💪",
+        address: liquidation.sponsor,
+        id: liquidation.id
       });
 
-      // Save all withdraw promises to resolve in parallel later on.
-      const withdrawPromises = [];
-      for (const liquidation of potentialWithdrawableLiquidations) {
-        Logger.info({
+      // Confirm that liquidation has eligible rewards to be withdrawn.
+      const withdraw = this.empContract.methods.withdrawLiquidation(liquidation.id, liquidation.sponsor);
+      let withdrawAmount;
+      try {
+        withdrawAmount = await withdraw.call({ from: this.account });
+      } catch (error) {
+        Logger.debug({
           at: "liquidator",
-          message: "attempting to withdraw rewards from liquidations💪",
+          message: "Liquidation failed or not ready for withdrawal",
           address: liquidation.sponsor,
           id: liquidation.id
         });
 
-        // Confirm that liquidation has eligible rewards to be withdrawn.
-        try {
-          let withdraw = this.empContract.methods.withdrawLiquidation(liquidation.id, liquidation.sponsor);
-          let withdrawAmount = await withdraw.call({ from: this.account });
-          if (this.web3.utils.toBN(withdrawAmount.rawValue).gt("0")) {
-            withdrawPromises.push(
-              withdraw.send({
-                from: this.account,
-                gas: 1500000,
-                gasPrice: this.gasEstimator.getCurrentFastPrice()
-              })
-            );
-          }
-        } catch (err) {
-          Logger.error({
-            at: "liquidator",
-            message: "failed to withdraw rewards from liquidation🤕",
-            address: liquidation.sponsor,
-            id: liquidation.id
-          });
-          continue;
-        }
+        continue;
       }
 
-      // Resolve all promises in parallel.
-      let promiseResponse = await Promise.all(withdrawPromises);
-
-      for (const response of promiseResponse) {
-        const logResult = {
-          tx: response.transactionHash,
-          caller: response.events.LiquidationWithdrawn.returnValues.caller,
-          withdrawalAmount: response.events.LiquidationWithdrawn.returnValues.withdrawalAmount,
-          liquidationStatus: response.events.LiquidationWithdrawn.returnValues.liquidationStatus
-        };
-        Logger.info({
-          at: "liquidator",
-          message: "withdraw tx result📄",
-          liquidationResult: logResult
-        });
-      }
-    } else {
-      Logger.debug({
+      Logger.info({
         at: "liquidator",
-        message: "No withdrawable liquidations"
+        message: `Will attempt to withdraw.`,
```

Comment:
nit: this info needs an emoji.
```suggestion
        message: `Will attempt to withdraw🤑`,
```